### PR TITLE
Updated weights, categories and added Marketing Cloud Email Specialist 

### PR DESCRIPTION
**Pull request by jesperklang**
Here I have:
- Fixed a bug that made it so that Sales Cloud couldn't be selected. It referenced Marketing Cloud Consultant.
- Added Marketing Cloud Email Specialist to the list of certifications
- Updated all the weights and categories for all certifications. The ones that needed updating was PD1, PD2, Marketing Cloud Consultant, Sales Cloud Consultant, Service Cloud Consultant and Identity and Access Management Designer

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -50,38 +50,71 @@ <h1 class="heading-1-dark">Salesforce Certification Checker</h1>
         </section>
 
         <section class="tool">
+            <div class="dropdown">
+                <!-- <select id="roles">
```
Comment (SCuvanov) [inline index.html]: Is it possible to drop this commented markup?

**Comment by SCuvanov** [comment]
I like the direction you are taking this project. Really good work. I left a comment about some commented code which I believe we can remove and then I will merge.

**Review by SCuvanov — COMMENTED** [review]
I like the direction you are taking this project. Really good work. I left a comment about some commented code which I believe we can remove and then I will merge.

File: index.html (hunk 1 above)
Comment (jesperklang) [reply index.html]: This is now fixed!

**Comment by jesperklang** [comment]
Thanks a lot for the feedback @SCuvanov! I work as a Certified Instructor and I got tired of maintaining and sending out Excel spreadsheets for every certifications for my students to calculate. Was very glad when a student directed me to your site. Great job!

**Pull request merged**